tests/core/creators/manual/detachedInline.md

```diff
@@ -0,0 +1,37 @@
+@bender-tags: 4.17.0, bug, 4462
+@bender-ui: collapsed
+@bender-ckeditor-plugins: toolbar,link,format,undo,floatingspace,sourcedialog
+
+
+1. Open dev console
```

Comment:
We ask tester to open console, but don't verify it at any point. We should either move this as a note at the beginning of the test with annotation to watch console for errors, or add it as a check in all `expected` clauses.

Same for all other manual tests.